.github/workflows/docker-manual.yml

```diff
@@ -13,16 +13,48 @@ on:
         required: true
 
 jobs:
-  publish:
+  publish_amd64:
+    name: Publish docker image for amd64
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@master
       with:
         submodules: recursive
         ref: "${{ github.event.inputs.ref }}"
     - name: Build the Docker image
-      run: docker build . --file Dockerfile --tag ${{ secrets.DOCKER_REPO }}:${{ github.event.inputs.tag }}
+      run: docker build . --file Dockerfile --tag ${{ secrets.DOCKER_REPO }}:${{ github.event.inputs.tag }}-amd64
     - name: Log into docker hub
       run: docker login --username ${{ secrets.DOCKER_USERNAME }} --password ${{ secrets.DOCKER_PASSWORD }}
-    - name: Push daily to Docker Hub
-      run: docker push ${{ secrets.DOCKER_REPO }}:${{ github.event.inputs.tag }}
+    - name: Push to Docker Hub
+      run: docker push ${{ secrets.DOCKER_REPO }}:${{ github.event.inputs.tag }}-amd64
```

Comment:
can we push to the same tag, just the different arch.

Reply:
We cannot, we have to push the tags separately and then combine them with manifest
We can delete them after manifest creation

Reply:
Let me try something